src/Permissions/Backend/PermissionManager.vala

```diff
@@ -36,95 +36,59 @@ public class Permissions.Backend.PermissionManager {
         _keys = new GenericArray<string> ();
         _values = new GenericArray<PermissionDescription> ();
 
-        insert ("shared=network", new PermissionDescription (
-            _("Network"),
-            _("App may be able to access other devices on the network"),
-            "preferences-system-network"
-        ));
-        insert ("shared=ipc", new PermissionDescription (
-            _("Communication between apps"),
-            _("App may be able to talk to, control and read information from other apps or system components"),
-            "internet-chat"
-        ));
-        insert ("sockets=x11", new PermissionDescription (
-            _("Graphical output"),
-            _("App may be able to show graphical windows"),
-            "video-display"
-        ));
-        //  insert ("sockets=fallback-x11", new PermissionDescription (
-        //      _("Access X11 windowing system (as fallback)"),
-        //      _("App is able to show graphical windows"),
-        //      "video-display"
-        //  ));
-        //  insert ("sockets=wayland", new PermissionDescription (
-        //      _("Access Wayland windowing system"),
-        //      _("App is able to show graphical windows"),
-        //      "video-display"
-        //  ));
-        insert ("sockets=pulseaudio", new PermissionDescription (
-            _("Sounds"),
-            _("App may be able to play sounds"),
-            "preferences-desktop-sound"
-        ));
-        insert ("sockets=system-bus", new PermissionDescription (
-            _("D-Bus system bus"),
-            _("App may be able to talk to, control and read information from system components via D-Bus"),
-            "internet-chat"
-        ));
-        insert ("sockets=session-bus", new PermissionDescription (
-            _("D-Bus session bus"),
-            _("App may be able to talk to, control and read information from other apps via D-Bus"),
-            "internet-chat"
-        ));
-        insert ("sockets=ssh-auth", new PermissionDescription (
-            _("Secure Shell agent"),
-            _("App may be able to access other devices on the network via SSH"),
-            "utilities-terminal"
-        ));
-        insert ("sockets=cups", new PermissionDescription (
-            _("Print"),
-            _("App may be able to access printers"),
-            "printer-printing"
+
+        insert ("filesystems=home", new PermissionDescription (
+            _("Home Folder"),
+            _("Access your entire home folder"),
```

Comment:
Do we know if this includes things like `.ssh` and other hidden folders? If so, it might be worth explicitly calling out here.

```suggestion
            _("Access your entire home folder, including any hidden folders"),
```